### DISCUSSION TITLE
Proposal: add link to source code repository to make it easier to get the changelog when doing a composer update

### DISCUSSION THREAD

**Discussion by jgroc-de** [composer/composer#11434] [General]
Hi,

I cant find a subject related to this so i open a new one.
When doing this with yarn `yarn upgrade-interactive`, you get a response like this: 
```
yarn upgrade-interactive v1.22.15
info Color legend : 
 "<red>"    : Major Update backward-incompatible updates 
 "<yellow>" : Minor Update backward-compatible features 
 "<green>"  : Patch Update backward-compatible bug fixes
? Choose which packages to update. (Press <space> to select, <a> to toggle all, 
? Choose which packages to update. 
re-js#readme
 ◯ postcss             latest  8.4.21  ❯  8.4.22  https://postcss.org/
 ◯ postcss-preset-env  latest  8.3.1   ❯  8.3.2   https://github.com/csstools/postcss-plugins/tree/main/plugin-packs/postcss-preset-env#readme
 ◯ webpack             latest  5.78.0  ❯  5.79.0  https://github.com/webpack/webpack
```

with the link to the website or the github. It makes it more easier to find the changelog and see what has changed (or see the commit or whatever).

Is it possible to add this when doing a `composer update` ?

**Comment by Seldaek** [comment]
`composer outdated` output has links to the package page if you ctrl-click the package name. That IMO is most comparable to what you are asking.

**Reply by jgroc-de** [reply]
ah yes nice, it does half the job but it is ok: no need to google each package to find the repo. Enough for now. I assume u do not want to add this? (i may try to do a PR if ever it seems interesting?)

**Reply by Seldaek** [reply]
Yeah I'd rather not add so much complexity in the update command as we already have some way to get links via outdated.

Note that as a related feature if you are interested in diffs/changelogs when you update packages we do offer that for PRs via https://packagist.com/features/update-review (it is a paid offering though, but it comes with more features).
